extensions/blocks/likes/likes-checkbox.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import { CheckboxControl } from '@wordpress/components';
+import { Component } from '@wordpress/element';
+import { compose } from '@wordpress/compose';
+import { withDispatch, withSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { __ } from '../../utils/i18n';
```

Comment:
Please merge this with external deps:

```suggestion
import { __ } from '@wordpress/i18n';
```